components/private_channel/rust/ffi/src/lib.h

```diff
@@ -0,0 +1,54 @@
+/* Copyright 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_COMPONENTS_PRIVATE_CHANNEL_RUST_FFI_SRC_LIB_H_
+#define BRAVE_COMPONENTS_PRIVATE_CHANNEL_RUST_FFI_SRC_LIB_H_
+
+#include <stdarg.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+#define C_KEY_SIZE 32
+
+typedef struct {
+  const uint8_t *encoded_partial_dec_ptr;
+  uintptr_t encoded_partial_dec_size;
+  const uint8_t *encoded_proofs_ptr;
+  uintptr_t encoded_proofs_size;
+  const uint8_t *random_vec_ptr;
+  uintptr_t random_vec_size;
+  bool error;
+} C_ResultSecondRound;
+
+typedef struct {
+  const uint8_t *pkeys_ptr;
+  uintptr_t pkeys_byte_size;
+  const uint8_t *skeys_ptr;
```

Comment:
`const uint8_t*` and don't use `_ptr` convention. It's not used in any chromium code

Reply:
- [ ] change from `uint8_t *` to `uint8_t*`
- [ ] remove `_ptr` from pointers names